package.json

```diff
@@ -31,9 +32,11 @@
     "@types/uuid": "^8.3.4",
     "@typescript-eslint/eslint-plugin": "^5.27.1",
     "@typescript-eslint/parser": "^5.27.1",
-    "eslint": "^8.17.0",
+    "chromedriver": "^107.0.3",
+    "eslint": "^8.27.0",
+    "mocha": "^10.1.0",
```

Comment:
I think `chromedriver` and `mocha` are not required in this project as a dependency.

Reply:
I'm using mocha to run the tests, in addition to nightwatch.

Reply:
At the moment there is only one test failing if you run it with mocha and I think that having mocha as an option to run the unit tests locally it's useful.